src/Controller.php

```diff
@@ -48,8 +49,15 @@ static function siteAction()
         $write = R::is('POST') and R::get('confirm');
         $site = site();
         $kirby = kirby();
+        $zipfile = kirby()->get('option', 'staticbuilder.zipfile', false);
```

Comment:
Instead of duplicating the logic for this, why not expose it as a public property or method of `$builder`?

This could limit the work in `siteAction` to just adding:

```php
return $builder->htmlReport([
  // ...
  'zipUrl' => $builder->zipUrl(),
  // ...
]);
```
(where the `zipUrl` methods returns false, or a URL string if the zip exists).

Would that be doable?